develop/devguide/Connector/DmsNotifyTypes/DmsNotifyOverview.md

```diff
@@ -130,3 +130,6 @@ The following table contains an overview of all Notify types.
 |117     |DMS_DISTRIBUTE_TRAP         |         |
```

Comment:
This file is not included in any TOC. Could you make sure it gets added somewhere?